packages/common/src/index.ts

```diff
@@ -754,8 +754,8 @@ export default class Common extends EventEmitter {
    * @param opts Hardfork options (onlyActive unused)
    * @return Array with hardfork arrays
    */
-  activeHardforks(blockNumber?: BNLike | null, opts: hardforkOptions = {}): Array<any> {
-    const activeHardforks = []
+  activeHardforks(blockNumber?: BNLike | null, opts: hardforkOptions = {}): HardforkParams[] {
```

Comment:
Just up for discussion, don't have a clear view myself: are/can be seen `any` -> `ConcreteType` typings as a breaking change (you just can do more nasty things with a return value typed as `any` and eventually (likely ?) people have done? 😀)?

Reply:
I think it depends on the context. In this case, since we're just really being more precise concerning the value that was already being returned, I feel like this is non-breaking. If we actually changed the return value, then I would agree it's breaking.